### PR TITLE
Issue #68: Added HotJar script to application layout and assets.rb

### PR DESCRIPTION
**Pull request by EChesters**
- No visual changes
- All tests ran and pass

### CODE REVIEW

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -9,3 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( hotjar.js )
```
Comment (steverabino) [inline config/initializers/assets.rb]: Change this to `Rails.application.config.assets.precompile += %w(hotjar.js)` to fix linting errors

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -14,3 +14,4 @@
         = content_tag :div, msg, class: [:alert, alert_for(name)]
     = yield
 
+    = javascript_include_tag 'hotjar.js'
```
Comment (steverabino) [inline app/views/layouts/application.html.haml]: Add new line after here to fix linting errors

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -14,3 +14,5 @@
...
+
```
Comment (steverabino) [inline app/views/layouts/application.html.haml:18]: This should really be a blank line (no whitespace)

**Comment by steverabino** [comment]
Apart from the linting, this unfortunately means very little to me!

@krissy @tanyapowell are you better placed to thumb?

**Comment by krissy** [comment]
LGTM 👍

**Pull request merged**